dockstore-client/src/main/java/io/dockstore/client/cli/nested/ApiClientExtended.java

```diff
@@ -16,16 +17,55 @@
 
 public class ApiClientExtended extends ApiClient {
 
+    /**
+     *
+     * @param key
+     * @return
+     */
+    private Optional<MediaType> getMediaType(Optional<String> key) {
+        // if key happens to be null or we don't recognize the key then return null
+        Optional<MediaType> mediaType = Optional.empty();
+        if (key.isPresent()) {
+            String keyWESParam = key.get().toLowerCase();
+            switch (keyWESParam) {
+            case "workflow_params":
```

Comment:
These strings seem fragile, but it doesn't look like there is an easy way to deal with this? What if a new parameter gets added - we'll need to remember to update this.